docusaurus.config.js

```diff
@@ -228,115 +228,6 @@ module.exports = async () => {
-                },
-                {
-                  from: '/shimmer/develop/nodes/core-rest-api',
-                  to: '/apis/core/v2',
-                },
-                {
-                  from: '/shimmer/goshimmer',
-                  to: '/goshimmer',
-                },
-                {
-                  from: '/shimmer/hornet',
-                  to: '/hornet/2.0.0-rc.6',
-                },
-                {
-                  from: '/shimmer/identity.rs',
-                  to: '/identity.rs/0.7',
-                },
-                {
-                  from: '/shimmer/introduction',
-                  to: '/introduction/stardust',
-                },
-                {
-                  from: '/shimmer/iota-sdk',
-                  to: '/iota-sdk',
-                },
-                {
-                  from: '/shimmer/iota.js',
-                  to: '/iota.js',
-                },
-                {
-                  from: '/shimmer/iota.rs',
-                  to: '/iota.rs/2.0.1-rc.7',
-                },
-                {
-                  from: '/shimmer/smart-contracts/guide/chains_and_nodes',
-                  to: '/smart-contracts/guide/chains_and_nodes',
-                },
-                {
-                  from: '/shimmer/smart-contracts/guide/core_concepts',
-                  to: '/learn/smart-contracts/core_concepts',
-                },
-                {
-                  from: '/shimmer/smart-contracts/guide/evm/compatibility',
-                  to: '/smart-contracts/guide/evm/compatibility',
-                },
-                {
-                  from: '/shimmer/smart-contracts/guide/evm',
-                  to: '/smart-contracts/guide/evm',
-                },
-                {
-                  from: '/shimmer/smart-contracts/guide/solo',
-                  to: '/smart-contracts/guide/solo',
-                },
-                {
-                  from: '/shimmer/smart-contracts/guide/wasm_vm',
-                  to: '/smart-contracts/guide/wasm_vm',
-                },
-                {
-                  from: '/shimmer/stronghold.rs',
-                  to: '/stronghold.rs',
-                },
-                {
-                  from: '/shimmer/team',
-                  to: '/team',
-                },
-                {
-                  from: '/shimmer/tutorials',
-                  to: '/tutorials',
-                },
-                {
-                  from: '/shimmer/wallet.rs',
-                  to: '/wallet.rs/1.0.0-rc.6',
-                },
-                {
-                  from: '/shimmer/learn/governance/',
-                  to: '/learn/governance/',
-                },
-              ];
-
-              for (const redirect of redirects) {
-                if (existingPath.includes(redirect.to)) {
-                  return existingPath.replace(redirect.to, redirect.from);
-                }
-              }
-            },
-          },
-        ],
```

Comment:
@Dr-Electron why does it remove the redirects? 🤔 something went wrong in the rebase?

Reply:
Fixed it by editing your commit